testsuite/integration/manualmode/src/test/config/arq/arquillian.xml

```diff
@@ -185,7 +185,7 @@
         <container qualifier="jbossas-messaging-ha-server1" default="false" mode="manual">
             <configuration>
                 <property name="jbossHome">${basedir}/target/jbossas-messaging-ha-server1</property>
-                <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/jbossas-messaging-ha-server1 -Djboss.node.name=default-jbossas-messaging-ha-server1</property>
+                <property name="javaVmArguments">${server.jvm.args} -Dboss.bind.address.private=${node0:127.0.0.1} -Djboss.default.multicast.address=${mcast} -Djboss.modcluster.multicast.address=${mcast} -Djboss.inst=${basedir}/target/jbossas-messaging-ha-server1 -Djboss.node.name=default-jbossas-messaging-ha-server1</property>
```

Comment:
BTW I am not a fan of those defaults here (the :127.0.0.1) part – the node0/1/2/4 are always set by the testsuite so this makes misconfig harder to debug.